introduction.md

```diff
@@ -1,6 +1,7 @@
 # dojo_rules
 
-Name = Aruna Sankpal
-Email = arunavs03@gmail.com
-Title = Software Engineer
-Favorite Code School Path = iOS
+* Name = Aruna Sankpal
+* Email = arunavs03@gmail.com
+* Title = Software Engineer
+* Favorite Code School Path = iOS
+* Deadly Skills = Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?